### DISCUSSION TITLE
Manual search gives unexpected results

### DISCUSSION THREAD

**Discussion by JohnAAbbott** [oscar-system/Oscar.jl#1915] [Q&A]
I was searching through the latest documentation (at [https://docs.oscar-system.org/stable/search/?q=pbw](url)
for the keyword `pbw`.  It produced many results, including several whose relevance is unclear to me.
For instance, why are there references to `pow`, `abs?   See the attached 
![Screenshot from 2023-02-09 14-49-29](https://user-images.githubusercontent.com/124266874/217838234-09f379c3-2fab-4f9a-adc9-1b46d7a54cae.png)
screenshot.

**Comment by fieker** [comment]
On Thu, Feb 09, 2023 at 06:21:00AM -0800, JohnAAbbott wrote:
> I was searching through the latest documentation (at [https://docs.oscar-system.org/stable/search/?q=pbw](url)
> for the keyword `pbw`.  It produced many results, including several whose relevance is unclear to me.
> For instance, why are there references to `pow`, `abs?   See the attached 
> ![Screenshot from 2023-02-09 14-49-29](https://user-images.githubusercontent.com/124266874/217838234-09f379c3-2fab-4f9a-adc9-1b46d7a54cae.png)
> screenshot.
> 
Trying to be helpful and accounting for spelling mistakes?

**Reply by fingolfin** [reply]
Yeah, Documenter's search function returns partially matches and things that "might sound similar" etc., too. At the same time, it is very limited in what it actually can search, and so it won't find all kinds of things. We've posted various issues about this to the Documenter.jl issue. If you want to know more about what is going on here specifically, I'd talk to them, too.
